.github/workflows/update-docs.yml

```diff
@@ -68,12 +68,6 @@ jobs:
       - name: Copy JS docs
         run: cp -r tauri/tooling/api/docs/* tauri-docs/docs/api/js/
         
-      - name: Copy Architecture doc
-        run: cp -r tauri/ARCHITECTURE.md tauri-docs/docs/about/
```

Comment:
shouldn't we keep this?

Reply:
No, we're going to do a link to this page, and the version on the site is going to be adapted to be more high-level. So for now I've copied it over so we have a starting point, but the goal of this and the one on the site will be different.

Reply:
ok